hooks/api/useFormLogin.js

```diff
@@ -25,19 +25,34 @@ export default function useFormLogin() {
    * @param {{token: string: user: object}} response Server response payload.
    */
   const login = async (response) => {
-    const { user, token } = response;
+    const {user, token} = response;
     const storage = storeToken(token); // Start IO.
     user.token = token;
     setUser(user);
     await storage; // Sync IO.
   };
+
+  /**
+   * 
+   * @param {sting} text 
+   */
+  const showToast = (text) => {
+    ToastAndroid.showWithGravity(
+      text,
+      ToastAndroid.SHORT,
+      ToastAndroid.CENTER
+    );
+  };
```

Comment:
Does this only work with Android devices or also with iPhones etc.? `react-native-toast-message` would be a better choice since it would work on both platforms
[https://www.npmjs.com/package/react-native-toast-message](https://www.npmjs.com/package/react-native-toast-message)